src/components/SourcesTree.js

```diff
@@ -161,6 +160,9 @@ class SourcesTree extends Component {
     }
 
     if (!nodeHasChildren(item)) {
+      if (this.props.sources.find(v => /react/.test(v))) {
```

Comment:
Here are the docs on [feature flags](https://github.com/devtools-html/debugger.html/blob/master/docs/feature-flags.md)